tests/ci.js

```diff
@@ -132,14 +132,16 @@ ls("tests/*.elm").forEach(function(testToRun) {
   } else if (/Failing\.elm$/.test(testToRun)) {
     echo("\n### Testing " + testToRun + " (expecting it to fail)\n");
     assertTestFailure(testToRun);
-  } else if (/PortRuntimeException.elm$/.test(testToRun)) {
+  } else if (/PortRuntimeException\.elm$/.test(testToRun)) {
+    echo("\n### TODO " + testToRun + " (Elm 0.19 beta allows multiple ports with the same name?)");
```

Comment:
TODO: figure out if this is a bug in Elm 0.19 beta

Reply:
-> yes, this is a bug in the compiler beta.  https://elmlang.slack.com/archives/C13L7S5GR/p1528239699000411